Tasks/ServiceFabricDeploy/task.json

```diff
@@ -83,6 +83,25 @@
             "groupname": "advanced",
             "helpMarkDown": "Indicates whether the application package should be compressed before copying to the image store. If enabled, this will override the value in the publish profile."
         },
+        {
+            "name": "useDiffPackage",
+            "type": "boolean",
+            "label": "Use Diff Package",
+            "defaultValue": "true",
+            "required": false,
+            "groupname": "advanced",
+            "helpMarkDown": "Upgrade by using a diff package that contains only the updated application files, the updated application manifest, and the service manifest files."
+        },
+        {
+            "name": "diffPackagePath",
```

Comment:
Can we use standard path like temp or sub-folder under working directory?

Reply:
@RoopeshNair We were thinking about giving a temp location as the default value for the path but we are not sure what is the value of the working directory assuming the preferred diffPackagePath is "WorkingDirectory/temp"